test/residency/prototypes/test_task015_hatchery_attribution_public_seam.py

```
@@ -0,0 +1,857 @@
+    ]
+
+
+EXPECTED_ROW_KEYS = {row.split("=", 1)[0] for row in expected_output_rows("linux")}
+
+
+def fail_unavailable() -> None:
+    print(FAILURE, file=sys.stderr)
+    raise SystemExit(1)
+
+
+def require(value: bool, message: str) -> None:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)